proposed/extended-coding-style-guide.md

```diff
@@ -216,13 +216,13 @@ For example:
 </html>
 ~~~
 
-Declare statements MUST contain no spaces and MUST be exactly `declare(strict_types=1)`
+Declare statements MUST be exactly `declare(strict_types = 1)`
```

Comment:
Shouldn't we make the semi-colon mandatory, so it follows the common rule about that too? At that point, this row can be changed into something like

> `declade` follows the same rules as the rest of the code: MUST have one space around the `=` operator and it MUST terminate with a semicolon

Reply:
I believe the mention of semicolon is only for cases like `<?php declare(strict_types = 1) ?>`. As shown in the example starting with "When wishing to declare...". Because otherwise it would be a parse error AFAIK. I don't think the case of semicolon before `?>` end tag is mentioned anywhere else. Maybe it should be removed from here and addressed separately ?